bedrock/mozorg/templates/mozorg/contribute/tasks/v1/build-firefox.html

```diff
@@ -1,28 +0,0 @@
-{# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
@craigcook we are going to do a big cleanup of the contribute/signup pages and everything that it links to so, perhaps we should exclude these from this PR as a lot of these pages will simply go away? Side note: as you have done the work, I am more than happy to merge it as is here so, up to you.